python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_functional_chaos_exploitation.py

```diff
@@ -70,27 +139,30 @@
 
 # %%
 # Get the multivariate basis
-# as a collection of Function
+# as a collection of :class:`~openturns.Function`.
 reduced = result.getReducedBasis()
 
 # %%
-# Get the orthogonal basis
+# Get the orthogonal basis.
 orthgBasis = result.getOrthogonalBasis()
 
 # %%
-# Get the distribution of variables Z
+# Get the standard distribution :math:`\mu` of the standardized
+# random vector :math:`\vect{Z}`.
 orthgBasis.getMeasure()
 
 # %%
-# Get the composed meta model which is the model of the reduced variables Z
-# within the reduced polynomials basis
+# Get the composed meta model :math:`\hat{h}` which is
+# the model of the standardized random vector :math:`\vect{Z}`
+# within the reduced polynomials basis.
 result.getComposedMetaModel()
```

Comment:
```suggestion
print(result.getComposedMetaModel())
```

Reply:
The same is true here. Without `print`:

![image](https://user-images.githubusercontent.com/31351465/236949956-e9ca4905-8732-42c6-85f6-7d0d469c387f.png)

With `print`:

![image](https://user-images.githubusercontent.com/31351465/236950042-58af29d2-25a7-47ab-9911-8fc7e0e8cda1.png)

The current code is better isn't it?

Reply:
@josephmure : If you agree on this particular point, I will resolve this one.

Reply:
However, if  #2282 improves the current code, then I agree to leave things as is.

Reply:
The C++ almost-pretty-print is at [UniVariatePolynomialImplementation.cxx](https://github.com/openturns/openturns/blob/457622ecf5264389f3739316e9c8cdc92aaea083/lib/src/Base/Func/UniVariatePolynomialImplementation.cxx#L75)